samples/bluetooth/peripheral_mds/README.rst

```diff
@@ -0,0 +1,176 @@
+After your development kits rebooting and reconnection with Bluetooth gateway it sends a coredump data to the Memfault cloud for further inspection and analysis.
+
+Memfault shell
+==============
+
+The Bluetooth MDS sample enables shell interface by default.
+You can use it to trigger different event instead of using development kit buttons.
+
+You can check the list of available command here: `Memfault Demo CLI`_.
+
+.. _peripheral_mds_user_interface:
+
+User interface
+**************
+
+The sample supports a simple user interface.
+You can control the sample using predefined buttons, while LEDs are used to display information.
+
+LED 1:
+   Blinks when the main loop is running (that is, the device is advertising) with a period of 2 seconds, duty cycle 50%.
+
+LED 2:
+   On when the development kit is connected.
+
+Button 1:
+   Press this button to start time measuring, the second pressing stops time measuring.
+
+   During the pairing procedure, press this button to accept the pairing.
+
+Button 2:
+   Triggers the ``Button2StateChanged`` trace event.
+
+   During the pairing procedure, press this button to reject the pairing.
+
+Button 3:
+   Every press of this button is counting under the ``Button3PressCount`` metric.
+
+Button 4:
+   Simulate development kit crash by triggering Hardfault exception by division by zero.
+
+
+Configuration
+*************
+
+The Memfault SDK allows the configuration of some of its options using Kconfig.
+To configure the options in the SDK that are not available for configuration using Kconfig, use :file:`samples/bluetooth/peripheral_mds/config/memfault_platform_config.h`.
+See `Memfault SDK`_ for more information.
+
+|config|
+
+To send data to the Memfault cloud through Bluetooth gateway, a project key must be configured using :kconfig:option:`CONFIG_MEMFAULT_NCS_PROJECT_KEY`.
+The additional required static configurations for this sample are following:
+
+   * :kconfig:option:`CONFIG_MEMFAULT_NCS_DEVICE_ID` - Memfault device id.
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/bluetooth/peripheral_mds`
+
+.. include:: /includes/build_and_run.txt
+
+Testing
+=======
+
+Before testing, ensure that your device is configured with the project key of your Memfault project.
+|test_sample|
+
+Testing with Memfault WebBluetooth Client
+-----------------------------------------
+
+1. |connect_terminal_ANSI|
+#. Reset development kit.
+#. Observe that the sample starts.
+#. Open or install a recent version of the `Google Chrome browser`_.
+#. Run the `Memfault WebBluetooth Client`_  script to forward memfault diagnostic data to the cloud.
+   For more details check the script source code here: `Memfault WebBluetooth CLient source code`_.
+#. Make sure that your development kit is advertising.
+#. In browser click :guilabel:`Connect` button and select your device from the list.
+#. Use buttons on a development kit to confirm or reject a pairing request.
```

Comment:
```suggestion
#. In the browser, click the :guilabel:`Connect` button and select your device from the list.
#. Use the buttons on your development kit to confirm or reject a pairing request:
```
Is the "browser" referring to the web browser? You mention "web browser" below.

Reply:
Yes. It is Google Chrome web browser